netbox/dcim/choices.py

```diff
@@ -929,6 +933,8 @@ class InterfaceTypeChoices(ChoiceSet):
         (
             'Ethernet (modular)',
             (
+                (TYPE_100ME_FX, '100BASE-FX (10/100ME FIBER)'),
+                (TYPE_100ME_LFX, '100BASE-LFX (10/100ME FIBER)'),
```

Comment:
These should probably go under the "fixed" group, as the standard applies to the physical transmission rather than the interface to a transceiver.

Reply:
That makes sense, I updated it